pages/index.tsx

```diff
@@ -475,21 +482,32 @@ export default function Home({ featuredStorefronts, selectedDaoSubdomains }: Hom
               Discover, explore, and collect NFTs from incredible creators on Solana
             </HeroTitle>
             <Pitch>Tools built by creators, for creators, owned by creators.</Pitch>
-            <Space direction="horizontal" size="large">
-              <WhiteRoundedButton onClick={() => router.push('/storefront/new')}>
-                Create Your Store
-              </WhiteRoundedButton>
-
-              {/* 
-               this is here as a refernce, can be removed at any point.
-               to allow free creation of marketplaces, go to /marketplace/new or uncomment the button :-)*/}
-
-              {MARKETPLACE_ENABLED && (
-                <WhiteRoundedButton onClick={() => router.push('/marketplace/new')}>
-                  Create Your Marketplace
-                </WhiteRoundedButton>
-              )}
-            </Space>
+            {connected ? (
+              <Space direction="horizontal" size="large">
+                {searching ? (
+                  <Skeleton.Button></Skeleton.Button>
```

Comment:
When the final button dimensions are in place this should be the same size and shape as the actual button so screen doesn't move while loading. @sjc5